caseconsulting/vue-expense-app
# POR 2614 - Tags Filtering into a seperate component

Ticket Link: [POR-2614](https://consultwithcase.atlassian.net/browse/POR-2614)

Made filtering into tags into a seperate component, along with an exported helper function, which is used in All the reports tabs, MyExpenses, Employees, Employees Export, PTO Cashouts, and Reinbursements.

How to test:
1. Reports tabs, go through all the reports tabs and check if filtering by tags and by negating the tags works
2. Same with MyExpenses, Employees, PTO Cashouts, and Reinbursements
3. For Employees Exporting Data, make sure that the exported spreadsheet is the same as if you filtered in Employees

[POR-2614]: https://consultwithcase.atlassian.net/browse/POR-2614?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
